util/deploy-to-maven-central.sh

```diff
@@ -60,4 +50,5 @@ for generated_pom_file in dagger*pom.xml; do
   rm "${generated_pom_file}.asc"
 done
 
-
+git tag dagger-"${version_name}"
+git push --tags origin master
```

Comment:
How about `git push origin tag "dagger-${version_name}"` to push only the just-created tag?